docs/monitors/appmesh.md

```diff
@@ -131,80 +135,41 @@ The following table lists the metrics available for this monitor. Metrics that a
 
 
-To specify custom metrics you want to monitor, add a `metricsToInclude` filter
-to the agent configuration, as shown in the code snippet below. The snippet
-lists all available custom metrics. You can copy and paste the snippet into
-your configuration file, then delete any custom metrics that you do not want
-sent.
 
-Note that some of the custom metrics require you to set a flag as well as add
-them to the list. Check the monitor configuration file to see if a flag is
-required for gathering additional metrics.
+This monitor does not send all metrics by default.  The metrics that are
+emitted without any non-required config options set are marked under the
+_Default_ column of the table above.
 
-```yaml
+### Non-default metrics (version 4.7.0+)
 
-metricsToInclude:
-  - metricNames:
-    - circuit_breakers.<priority>.cx_open
-    - circuit_breakers.<priority>.cx_pool_open
-    - circuit_breakers.<priority>.remaining_cx
-    - circuit_breakers.<priority>.remaining_pending
-    - circuit_breakers.<priority>.remaining_retries
-    - circuit_breakers.<priority>.remaining_rq
-    - circuit_breakers.<priority>.rq_open
-    - circuit_breakers.<priority>.rq_pending_open
-    - circuit_breakers.<priority>.rq_retry_open
-    - external.upstream_rq_<_>
-    - external.upstream_rq_<_xx>
-    - external.upstream_rq_completed
-    - external.upstream_rq_time
-    - internal.upstream_rq_<_>
-    - internal.upstream_rq_<_xx>
-    - internal.upstream_rq_completed
-    - internal.upstream_rq_time
-    - membership_change
-    - membership_degraded
-    - upstream_cx_active
-    - upstream_cx_close_notify
-    - upstream_cx_connect_attempts_exceeded
-    - upstream_cx_connect_fail
-    - upstream_cx_connect_ms
-    - upstream_cx_connect_timeout
-    - upstream_cx_destroy
-    - upstream_cx_destroy_local
-    - upstream_cx_destroy_local_with_active_rq
-    - upstream_cx_destroy_remote
-    - upstream_cx_destroy_remote_with_active_rq
-    - upstream_cx_destroy_with_active_rq
-    - upstream_cx_http1_total
-    - upstream_cx_http2_total
-    - upstream_cx_idle_timeout
-    - upstream_cx_length_ms
-    - upstream_cx_max_requests
-    - upstream_cx_none_healthy
-    - upstream_cx_overflow
-    - upstream_cx_pool_overflow
-    - upstream_cx_protocol_error
-    - upstream_cx_rx_bytes_buffered
-    - upstream_cx_total
-    - upstream_cx_tx_bytes_buffered
-    - upstream_cx_tx_bytes_total
-    - upstream_rq_active
-    - upstream_rq_cancelled
-    - upstream_rq_maintenance_mode
-    - upstream_rq_pending_active
-    - upstream_rq_pending_failure_eject
-    - upstream_rq_pending_overflow
-    - upstream_rq_pending_total
-    - upstream_rq_per_try_timeout
-    - upstream_rq_retry_overflow
-    - upstream_rq_rx_reset
-    - upstream_rq_timeout
-    - upstream_rq_total
-    - upstream_rq_tx_reset
-    monitorType: appmesh
-```
+**The following information applies to the agent version 4.7.0+ and has
```

Comment:
Run-on; need to break out into a bullet list of phrases separated by semi=colons.  like " The following information assumes: You are using agent version 4.7.0 or higher; you have set... to true (link to where that's documented); and there are no references to .... in metricsToExclude..."